torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -460,6 +460,9 @@ at::Tensor XLANativeFunctions::_copy_from(const at::Tensor& self,
                                           const at::Tensor& dst,
                                           bool non_blocking) {
   TORCH_LAZY_FN_COUNTER("xla::");
+  XLA_CHECK(self.device().type() == c10::kCPU ||
```

Comment:
hmm, so I guess this is useful when someone tries to copy a cuda tensor to xla device?

Reply:
yes, currently torch_xla will die with core dump due to illegal memory access if a user tries to do so.